### PR TITLE
Add a blog post describing how to create a bootable jar for a web application secured using Keycloak.

### PR DESCRIPTION
**Pull request by darranl**
Just placing this one on hold as well whilst we wait for a Keycloak release to publish this.

This blog depends on the following example https://github.com/wildfly-security-incubator/elytron-examples/pull/39

### CODE REVIEW

**Comment by darranl** [comment]
@bstansberry / @jfdenise FYI this is the second blog post, also ready to go out once KeyCloak 12 is released.

File: _posts/2020-10-21-bootable-keycloak.adoc [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+----
+<plugin>
+    <groupId>org.wildfly.plugins</groupId>
+    <artifactId>wildfly-jar-maven-plugin</artifactId>
+    <version>${version.wildfly.jar.maven.plugin}</version>
+    <configuration>
+        <feature-packs>
+            <feature-pack>
+                <location>wildfly@maven(org.jboss.universe:community-universe):current</location>
+                <inherit-packages>false</inherit-packages>
```
Comment (jfdenise) [inline _posts/2020-10-21-bootable-keycloak.adoc:74]: Not needed, the default value is false.

File: _posts/2020-10-21-bootable-keycloak.adoc [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+            </feature-pack>
+            <feature-pack>
+                <groupId>org.keycloak</groupId>
+                <artifactId>keycloak-adapter-galleon-pack</artifactId>
+                <version>${version.keycloak}</version>
+                <inherit-packages>false</inherit-packages>
```
Comment (jfdenise) [inline _posts/2020-10-21-bootable-keycloak.adoc:80]: Not needed, the default value is false.

File: _posts/2020-10-21-bootable-keycloak.adoc [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+
+Firstly two feature packs are defined, one is defined to access the WildFly feature pack and a second is defined to access the Keycloak
+adapter feature pack.
+
+[source,xml]
+----
+<feature-packs>
+    <feature-pack>
+        <location>wildfly@maven(org.jboss.universe:community-universe):current</location>
+        <inherit-packages>false</inherit-packages>
```
Comment (jfdenise) [inline _posts/2020-10-21-bootable-keycloak.adoc:116]: Not needed, the default value is false.

File: _posts/2020-10-21-bootable-keycloak.adoc [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+    </feature-pack>
+    <feature-pack>
+        <groupId>org.keycloak</groupId>
+        <artifactId>keycloak-adapter-galleon-pack</artifactId>
+        <version>${version.keycloak}</version>
+        <inherit-packages>false</inherit-packages>
```
Comment (jfdenise) [inline _posts/2020-10-21-bootable-keycloak.adoc:122]: Not needed, the default value is false.

**Comment by jfdenise** [comment]
@darranl , thank-you. Very good, I added a comment for a configuration attribute that you can omit.

**Comment by darranl** [comment]
Thank you @jfdenise I have removed those lines.

**Comment by fjuma** [comment]
Because this PR was about the previous keycloak-adapter-galleon-pack which no longer exists, I'm going to close this one.

**Pull request closed without merging**